node.js

```diff
@@ -46,7 +46,8 @@ const keys =
     "    B9E2F5981AA6E0CD28160D9FF13993A75599653C ", // Shelley Vohr <shelley.vohr@gmail.com>, Node release team
     "    C82FA3AE1CBEDC6BE46B9360C43CEC45C17AB93C ", // Richard Lau <riclau@uk.ibm.com>, Node release team
     "    108F52B48DB57BB0CC439B2997B01419BD92F80A ", // Ruy Adorno <ruyadorno@hotmail.com>, Node release team
-    "    1C050899334244A8AF75E53792EF661D867B9DFA ", // Danielle Adams <adamzdanielle@gmail.com>, Node release team
+    "    1C050899334244A8AF75E53792EF661D867B9DFA ", // Danielle Adams <adamzdanielle@gmail.com>, Node release team, prior to 01/09/21
```

Comment:
I'm not sure if this old key should be removed yet. I'm going to check if any releases were signed with it.

Reply:
It appears both https://nodejs.org/en/blog/release/v15.2.0/ and https://nodejs.org/en/blog/release/v15.4.0/ were signed with this old key.